h2o-mapreduce-generic/src/main/java/water/hadoop/DelegationTokenRefresher.java

```diff
@@ -20,37 +21,47 @@
 
 public class DelegationTokenRefresher implements Runnable {
   
-  public static final String KEYTAB_FILE = "hive_keytab";
-  private static final String KEYTAB_PATH = "./" + KEYTAB_FILE;
-
-  public static void setup(Configuration conf) {
+  public static void setup(Configuration conf, String iceRoot) throws IOException {
```

Comment:
do we need to worry about creating the iceRoot? The only place where this is used is in the h2omapper which will make the directory for us - I would rather simplify and remove the call of `makeSureIceRootExists` + rename iceRoot to something else, eg. keytabDirectory

Reply:
ice_root is created only when any H2O_MAPPER_CONF_* vars are set, so yes, we need to make sure it exists